modules/models/src/main/java/org/eclipse/xpanse/modules/models/security/model/CurrentUserInfoHolder.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.models.security.model;
+
+import java.util.Objects;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * User information holder.
+ */
+@Profile("zitadel")
```

Comment:
do we need this profile annotation as this class is not a spring bean?

please also consider that if Zitdel profile is not enabled .. then there is no token available. So in this case, no auth header must be sent.